File: ACC/HikAGVWebAPI/HikAGVWebAPI/Models/oPortModel.cs
public class oPortModel
{
    public string Area { get; set; }
    public string Block { get; set; }
    public string Port { get; set; }
    public string StationNo { get; set; }
    public string InterfaceName { get; set; }
    public string PanelCallShuttle { get; set; }
    public string Priority { get; set; }
    public string ProductionPartNo { get; set; }
    public string UseFlag { get; set; }
    public string RackId { get; set; }
    public string WorkOrder { get; set; }
    public string HaveFlag { get; set; }
    public string Remark { get; set; }
    public string PutTime { get; set; }
    public string BgnToEnd { get; set; }
    public override string ToString()
    {
        return $@"[Area] : {Area}; [Block] : {Block}; [Port] : {Port}; [Station No] : {StationNo}; [Interface Name] : {InterfaceName}; [Panel Call Shuttle] : {PanelCallShuttle}; [Priority] : {Priority}; [Production Part No] : {ProductionPartNo}; [Use Flag] : {UseFlag}; [Rack ID] : {RackId}; [Work Order] : {WorkOrder}; [Have Flag] : {HaveFlag}; [Remark] : {Remark}; [Put Time] : {PutTime}; [Begin To End] : {BgnToEnd}";
    }
}

File: Android/saa/app/src/main/java/com/example/saa/oPortModel.kt
package com.example.saa

import android.os.Parcel
import android.os.Parcelable

data class oPortModel(
    var Area: String,
    var Block: String,
    var Port: String,
    var StationNo: String,
    var InterfaceName: String?,
    var Priority: String,
    var UseFlag: String,
    var RackID: String?,
    var WorkOrder: String?,
    var HaveFlag: String?,
    var BgnToEnd: String?,
    var MachineName: String?,
    var PutTime: String? =null,
): Parcelable {
    constructor(parcel: Parcel) : this(
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
        parcel.readString().toString(),
    )

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(Area)
        parcel.writeString(Block)
        parcel.writeString(Port)
        parcel.writeString(StationNo)
        parcel.writeString(InterfaceName)
        parcel.writeString(Priority)
        parcel.writeString(UseFlag)
        parcel.writeString(RackID)
        parcel.writeString(WorkOrder)
        parcel.writeString(HaveFlag)
        parcel.writeString(BgnToEnd)
        parcel.writeString(MachineName)
        parcel.writeString(PutTime)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<oPortModel> {
        override fun createFromParcel(parcel: Parcel): oPortModel {
            return oPortModel(parcel)
        }

        override fun newArray(size: Int): Array<oPortModel?> {
            return arrayOfNulls(size)
        }
    }
}